README.md

```diff
@@ -298,6 +298,9 @@ Default:
   "azure_event_grid": {
     "zone_name": "privatelink.eventgrid.azure.net"
   },
+  "azure_event_grid_mqtt": {
```

Comment:
Should the internal key reflect the purpose? 
In this case the zone is referring to "topicspace", in short  ".ts"."

But this feature is actually only available for MQTT-enabled Event Grids.

So: The internal key could be "azure_event_grid_mqtt" or "azure_event_grid_topicspace".

What do you prefer?